openmp/runtime/src/z_Linux_util.cpp

```diff
@@ -878,6 +878,15 @@ void __kmp_create_worker(int gtid, kmp_info_t *th, size_t stack_size) {
     KMP_SYSFAIL("pthread_create", status);
   }
 
+#ifdef _GNU_SOURCE
```

Comment:
Is it a convention to check _GNU_SOURCE before __GLIBC__? Feels a bit unnecessary..

Also wondering if there is a more portable way of doing this.